Sources/StreamChatUI/Gallery/GalleryVC.swift

```diff
@@ -342,10 +342,12 @@ open class GalleryVC: _ViewController,
             return UICollectionViewCell()
         }
         
-        let cell = collectionView.dequeueReusableCell(
+        guard let cell = collectionView.dequeueReusableCell(
             withReuseIdentifier: reuseIdentifier,
             for: indexPath
-        ) as! GalleryCollectionViewCell
+        ) as? GalleryCollectionViewCell else {
+            return UICollectionViewCell()
+        }
```

Comment:
The same, would generic dequeue work here as well?

Reply:
Hmm good spot, so this also changes things a little because we change cell type based on the index path contents, I can introduce an optional param to the reusable func that takes the reuseId in these situations, WDYT?